base/debian/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM debian:bullseye-20220509-slim
```

Comment:
any reason to lock the version here?

Reply:
Yeah, so nothing breaks in future. There's no downside to this but if you want I can remove that.

Reply:
I think locking it makes sense. Better to be exact about which version to use.